witness/src/Dscp/Witness/Web/API.hs

```diff
@@ -82,6 +82,11 @@ data WitnessEndpoints route = WitnessEndpoints
         :- "checkcv"
         :> ReqBody '[JSON] FairCV
         :> Verb 'PUT 200 '[DSON] FairCVCheckResult
+
+    , wCheckFairCVPDF :: route
+        :- "checkcv"
+        :> ReqBody '[OctetStream] ByteString
+        :> Verb 'PUT 200 '[DSON] (FairCVCheckResult, FairCV)
```

Comment:
This response would look like a JSON-list from frontend's point of view, and it's supposed canonical to always return JSON-objects. Better create a datatype for this thing (we do so in every API).